src/main/java/racing/Car.java

```diff
@@ -1,50 +1,90 @@
 package racing;
 
-import java.util.ArrayList;
-import java.util.List;
 import java.util.Random;
 
 /**
  * 초간단 자동차 경주 게임의 자동차 개체 하나를 나타냅니다.
  */
 public class Car {
-    /** 자동차가 움직이는 조건을 판단하는 데에 쓰이는 조건 판별 기준 상수 */
-    private static final int MOVE_CONDITION = 4;
+    /** 사용자가 위치를 지정하지 않았을 때 기본 시작할 자동차의 위치 */
+    private static final int DEFAULT_INIT_POSITION = 1;
+
+    /** 허용하는 최대 이름 글자 수 */
+    public static final int MAX_NAME_LENGTH = 5;
 
     /** 자동차의 현재 위치를 나타냅니다. */
     private int position = 0;
+
+    /** 자동차의 이름 */
+    private String name = "";
+
+
     private final Random random = new Random();
 
+    private Car() {
+    }
+
     /**
-     * 초기 위치를 지정하여 자동차를 생성합니다.
-     * @param initPosition 초기 위치. 0 이상이여야 합니다.
+     * 가장 기본적인 자동차를 생성합니다.
+     * 익명 이름에 기본 위치로 지정됩니다.
+     *
+     * @return 생성된 자동차 객체
      */
-    public Car(int initPosition) {
-        validatePosition(initPosition);
+    public static Car create() {
+        Car car = new Car();
```

Comment:
```suggestion
    public static Car create() {
        Name name = Name.anonymous();
        Position position = Position.zero();
        
        return new Car(name, position);
```

일급 객체를 사용해서 좀 더 가독성을 높혀볼 수 있을 것 같네요. 
단순 공백을 할당한다는건 의도를 파악하기 힘들 수 있으니까요.

Reply:
오... 이 생각은 못 해봤는데 좋네요.